message_ix_models/tools/iea/weo.py

```diff
@@ -0,0 +1,672 @@
+"""
+Code for handling IEA WEO data
+"""
+
+import numpy as np
+import pandas as pd
+
+from message_ix_models.util import package_data_path
+
+
+def get_weo_data():
+    """
```

Comment:
For docstrings, per the [code style](https://docs.messageix.org/en/stable/contributing.html#code-style), we follow the numpydoc standard. See the links there to the standard, PEP 257, etc.